airiusc/policy-management
# handled different client creation errors

This one's a bit harder to test:

1) start front-end and back-end servers
2) change the copyFrom folder ID and make a submit front-end questionnaire and see if you get something indicating the template folder wasn't found
3) when you finish 2, do a ctrl-s in a back-end file to restart the back-end server. Then, without refreshing the browser front-end, submit the questionnaire and see if you get something like "user isn't authorized"
